exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -373,6 +381,9 @@ func addSingleHistogramDataPoint(pt pdata.HistogramDataPoint, resource pdata.Res
 			Value:     float64(cumulativeCount),
 			Timestamp: time,
 		}
+		if pt.Flags().HasFlag(pdata.MetricDataPointFlagNoRecordedValue) {
+			bucket.Value = math.Float64frombits(value.StaleNaN)
```

Comment:
I would expect that this would never be reached as a histogram with no recorded value shouldn't have any buckets either, but since I believe that Prometheus will handle this correctly on receipt if it does happen this is probably fine.